string link
int8 value
---
int8 resultCode